### ISSUE TITLE
Hello,I cant run this project

### ISSUE THREAD

**Issue by 22w2w2** [Js-Mim/mss_pytorch#7]
My mac dont have GPU,so how I cant run this project.
Can I close GPU and use CPU to run this project?
Your project is very worthwhile for me to learn!
By the way,this project has tensorflow Edition？

**Comment by Js-Mim** [comment]
Hi @22w2w2,

i am able to run this project on my MacBook without any GPU.
Installing "Anaconda" and "Pytorch" is necessary, though. You can find much information online on
how to install these.
After you have installed the necessary dependencies, let me know if it works for you.

**Comment by 22w2w2** [comment]
Thank you very much for your reply, I have solved most of the problems, but there is a little bug, so I am sorry to disturb you again.
1.mixtures_path = '/home/avdata/audio/own/dsd100/DSD100/Mixtures/'
sources_path = '/home/avdata/audio/own/dsd100/DSD100/Sources/'
Where can I download these wav

2.window = viz.line(X=np.arange(0, 1),
                      Y=np.reshape(0, 1),
                      opts=dict(
                      fillarea=True,
                      # legend=False,
This parameter legend I have to delete it，Otherwise it will be BUG,If you do not have this bug, let me slowly modify it.
I am a Chinese graduate who admires your team very much, thank you very much for your reply

**Comment by Js-Mim** [comment]
Hey, you are very welcome!

1. You need to download some data in order  to fit the model. We used DSD100 and some mixtures from the MedleydB datasets. You will find much information online, on how you can obtain them. After downloading the data, please change these two paths to the paths that your downloaded data is stored. If i recall correctly from your previous comment, you are willing to use that code without a GPU. The amount of data combined with the recurrent inference over gated, recurrent units it's gonna take you forever! I would suggest you writing your own data handling function and fine tune the weights (see here https://js-mim.github.io/mss_pytorch/) using a couple of training examples.

2. I apologize for that. I have forgotten to mention my torchvision version, which is outdated. This will be fixed.

**Comment by 22w2w2** [comment]
The project now works perfectly, but after the Ideal Amplitude Ratio Mask, line 39 in nnet_helpers.py ms = np.pad (ms, ((0, trim_frame), (0, 0)), 'constant', constant_values = (0, 0)),
a bug has occurred： Unable to create correctly tuple from ((0, 40), (0, 0))
I know that by removing it it is possible to bypass the bug and continue to run, but I have to solve this problem, ha ha, bother you again (· ω ·)

**Comment by Js-Mim** [comment]
I cannot reproduce the provided error.
Could you please check the shape of the "ms" argument.
Numpy pad, will simply add some 0 dimensional vectors that are going to be
segmented to create magnitude time-sequences.
